### PR TITLE
[anu]#11038 Fixed es-lint issues

### PR DESCRIPTION
**Pull request by anusreePrakash**
- removed ghost/mocha/no-sibling-hooks

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [ ] There's a clear use-case for this code change
- [ ] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test` and `yarn lint`)

More info can be found by clicking the "guidelines for contributing" link above.

### CODE REVIEW

File: core/test/acceptance/old/admin/themes_spec.js [hunk 1]
```diff
@@ -22,17 +22,13 @@ describe('Themes API', function () {
             .set('Origin', config.get('url'))
             .attach(fieldName, themePath);
     };
-
     before(function () {
-        return ghost()
```
Comment (allouis) [inline core/test/acceptance/old/admin/themes_spec.js]: `mocha` uses the return value (if it's a promise) in `before`, `beforeEach` etc.. to know when the setup/teardown has completed. So remove of the `return` here is changing functionality. I think you'll want to either update these to `async` functions and `await` the operations, continue to `return` promises OR change it to `before(function (done) {` and call `done` when everything is setup!

**Review by allouis — CHANGES_REQUESTED** [review]

File: core/test/acceptance/old/admin/themes_spec.js [hunk 2]
```diff
@@ -23,259 +23,262 @@ describe('Themes API', function () {
...
-                jsonResponse.themes[0].name.should.eql('valid');
-                jsonResponse.themes[0].active.should.be.false();
-
-                // ensure tmp theme folder contains two themes now
-                const tmpFolderContents = fs.readdirSync(config.getContentPath('themes'));
-                tmpFolderContents.forEach((theme, index) => {
-                    if (theme.match(/^\./)) {
-                        tmpFolderContents.splice(index, 1);
-                    }
+    describe('fafa', function () {
```
Comment (gargol) [inline core/test/acceptance/old/admin/themes_spec.js]: Not sure this is a good name for this describe group :grimacing:

**Comment by gargol** [comment]
:wave:  @anusreePrakash let us know if you need help with these changes. I'd focus on only fixing the `no-sibling-hooks` rule for now and do any additional improvements to test suites as a separate PR (regarding changes like this onehttps://github.com/TryGhost/Ghost/pull/11178#discussion_r329880611)

**Comment by gargol** [comment]
Happy to reopen whenever you have some time to finish the PR @anusreePrakash :wink:

**Pull request closed without merging**